sbt/src/sbt-test/typelevel/scala-organization/build.sbt

```diff
@@ -0,0 +1,8 @@
+organization := "org.blah"
+
+//scalaOrganization := "org.scala-lang"
+scalaOrganization := "com.milessabin"
```

Comment:
> How should this be tested? 

For scripted tests, I'm ok with creating dependencies to external repository configuration. We have many examples of that in `dependency-management/` group of scripted tests.

> Should this be done via a mock repository? Is there an example that I could base tests on in the tree?

If you grep for `"repo"` you'll find examples of us creating fake repos like this one:
https://github.com/sbt/sbt/tree/bb332fae594c7393f5aa545ff6a56cd8abefc9c2/sbt/src/sbt-test/dependency-management/cached-resolution-configurations
But that would require checking scala-compiler, scala-library JARs, so..